base/cvd/cuttlefish/host/libs/zip/zip_cc.cc

```diff
@@ -504,15 +504,20 @@ Result<SeekableZipSource> ReadableZip::GetFile(uint64_t index) {
 
 ReadableZip::ReadableZip(std::unique_ptr<Impl> impl) : impl_(std::move(impl)) {}
 
-Result<WritableZip> WritableZip::FromSource(WritableZipSource source) {
+Result<WritableZip> WritableZip::FromSource(WritableZipSource source,
+                                            bool truncate) {
```

Comment:
Can the public interface of this either use an `enum` or be two functions, like `OpenSource` and `TruncateSource`?

See https://abseil.io/tips/94

Reply:
Done